AcceptanceTests/runAcceptanceTests.sh

```diff
@@ -45,23 +45,30 @@ echo "Running Muter on an empty example codebase..."
 cd ./Repositories/EmptyExampleApp
 
 echo " > Running in CLI mode..."
-../../.build/x86_64-apple-macosx/debug/muter > ../../AcceptanceTests/muters_empty_state_output.txt
+../../.build/x86_64-apple-macosx/debug/muter &> ../../AcceptanceTests/muters_empty_state_output.txt
 cd ../..
 
 echo "Running Muter on an example test suite that fails..."
 cd ./Repositories/ProjectWithFailures
 
 echo " > Running in CLI mode..."
-../../.build/x86_64-apple-macosx/debug/muter > ../../AcceptanceTests/muters_aborted_testing_output.txt
+../../.build/x86_64-apple-macosx/debug/muter &> ../../AcceptanceTests/muters_aborted_testing_output.txt
 rm -rf ./muter_logs # don't pollute the staging area
 
 cd ../..
 
 echo "Running Muter's help command..."
 cd ./Repositories/ExampleApp
 
-echo " > Running command..."
-../../.build/x86_64-apple-macosx/debug/muter help > ../../AcceptanceTests/muters_help_output.txt
+echo " > Running help command..."
+../../.build/x86_64-apple-macosx/debug/muter help &> ../../AcceptanceTests/muters_help_output.txt
```

Comment:
can we make a variable like `MUTER_PATH="../../build/x86_64-apple-macosx/debug/muter"`? (Missing `.` in front of `build` is intentional. It's not needed, AFAIK.